cli/src/pcluster/utils.py

```diff
@@ -563,10 +563,7 @@ def get_supported_os_for_scheduler(scheduler):
 
 def get_supported_os_for_architecture(architecture):
     """Return list of supported OSes for the specified architecture."""
-    oses = ["alinux2", "ubuntu1804", "ubuntu2004", "centos8"]
-    if architecture == "x86_64":
-        oses.extend(["centos7"])
-    return oses
+    return ["alinux2", "ubuntu1804", "ubuntu2004", "centos7", "centos8"]
```

Comment:
We can just remove `get_supported_os_for_architecture` function and use `SUPPORTED_OSS` everywhere the function is used.

`architecture_os_validator` and `_validate_createami_args_ami_compatibility` logic might need to be modified to account for the warning message when using centos7+arm.

Reply:
Ok fixed!